dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/h2/V6.0_2017.07.04__DS-3636_General_Index_Performance_Updates.sql

```diff
@@ -0,0 +1,24 @@
+--
+-- The contents of this file are subject to the license and copyright
+-- detailed in the LICENSE and NOTICE files at the root of the source
+-- tree and available online at
+--
+-- http://www.dspace.org/license/
+--
+
+------------------------------------------------------
+-- DS-3636 General index and caching perfomance fixes
+------------------------------------------------------
+-- Indexes on resourcepolicy, versionitem, metadata and bundle2bitstream colums to improve searching efficiency
+
+DROP INDEX IF EXISTS resourcepolicy_action_idx;
+DROP INDEX IF EXISTS resourcepolicy_resource_type_id_idx;
+DROP INDEX IF EXISTS versionitem_versionhistory_id_idx;
+DROP INDEX IF EXISTS bundle2bitstream_bitstream_order_idx;
+DROP INDEX IF EXISTS metadatavalue_mf_place_idx;
+
+CREATE INDEX resourcepolicy_action_idx ON resourcepolicy (action_id);
```

Comment:
This field should not be indexed on its own.  There would never be a need to retrieve items by action_id alone.